test/Elastic.Apm.AspNetCore.Tests/Helper.cs

```diff
@@ -32,7 +32,7 @@ public static class Helper
 						app.UseStaticFiles();
 						app.UseCookiePolicy();
 
-#if NETCOREAPP3_0
+#if NETCOREAPP3_0 || NETCOREAPP3_1
```

Comment:
I thought about it when I was working for adding the `netcoreapp3.0` target, and it seems it's also relevant for the `netcoreapp3.1` target. What you think about extracting routing/mvc registration to a separate method?

Reply:
> What you think about extracting routing/mvc registration to a separate method?

Makes sense, we repeated this code a lot. I moved it into a separate method.